var/spack/repos/builtin/packages/opa-psm2/package.py

```diff
@@ -29,10 +29,18 @@ class OpaPsm2(MakefilePackage):
     # patch to prevent opa-psm2 from adding an additional "usr/"
     #   subdirectory within the installation prefix, which breaks paths for
     #   dependent packages like libfabric
-    patch('opa-psm2-install-prefix.patch', when='@11.2.68:') 
+    patch('opa-psm2-install-prefix.patch', when='@11.2.68:')
+
+    # patch to get the Makefile to use the spack compiler wrappers
+    patch('opa-psm2-compiler.patch', when='@11.2.68:',
+          sha256='fe31fda9aaee13acb87d178af2282446196d2cc0b21163034573706110b2e2d6')
```

Comment:
I don't think checksums are required for local patches, but it might be fine to add them.